assignment-1/Failed__case.md

```diff
@@ -1,87 +1,86 @@
-
-
-
-#### Sources
-
-[1] Cleland-Huang, Jane. "Don't Fire the Architect! Where Were the Requirements?." Software, IEEE 31.2 (2014): 27-29.
-
-[2] Morgan, David &  Humer, Caroline. "Timeline: U.S. healthcare law's technology breakdown" Reuters, 30 October 2013, Web. 9 February 2015.
-
-[3] Levinson, Daniel. "CMS Did Not Always Manage and Oversee Contractor Peformance for the Federal Marketplace as Required by Federal Requirements and Contract Terms", Office of Inspector General, September 2015, Report.
-
-[4] Anthopoulos, L., et al., Why e-government projects fail? An analysis of the Healthcare.gov website, Government Information Quarterly (2015), http://dx.doi.org/10.1016/j.giq.2015.07.003
-
-[5] Government Accountability Office (GAO) (2013). “Patient protection and affordable care act: Status of CMS efforts to establish federally facilitated health insurance exchanges.” (GAO-13-601), 19 June 2013.
-
-[6] Tom Gilb (Planguage). (2001) Intel Corporation "Quantifying Quality Requirements Using Planguage" by Erik Simmons.
-
+## Failed case : HealthCare.gov
+
+##### Initial complexity.
+
+The construction of healthcare.gov was overseen by Centers for Medicare and Medicaid Services (CMS) which is a part of the US Department of Health and Human Services (HHS). The primary contractor was CGI who had a contract of around 250 million dollars. Next to that there were 16 official subcontractors, but the total number of subcontractors was actually 55. All under the supervison of CMS. There were also 300 private insurers. All in all there were around 4000 plans.
+
+The client were according to [4, page 6]: the Department of Health and Human Services (HHS), Centers for Medicare & Medicaid Services (CMS), 36 States, 300 private insurers, U.S. Chief Technology Office, GAO, Media, Citizens, Social Security Administration, the Internal Revenue Service, Veterans Administration, Office of Personnel Management, Peace Corps, etc.
+
+The internal architecture of healthcare.gov is very complex. US residents who want to apply for an insurance needed a lot of information from the different systems. Like their income and immigration status. The website needed to be connected with all the different systems from the federal government and the private insurers. Like the Internal Revenue Service, Social Security Administration, the Peace Corps, etcetera. The idea was the information of all the different federal systems should be retrieved real time. According to [4] the U.S. Chief Technology Officer Todd Park has said that the government expected that the government expected HealthCare.gov to draw 50,000 to 60,000 simultaneous users. But in the first week the site was overwhelmed by up to five times as many users. 
```

Comment:
etcetera could be abbreviated to etc. -or- be consistent and change other appearances throughout the text.